File: recycler-view-basic/app/src/main/java/com/sqisland/android/espresso/recycler_view_basic/TextViewHolder.java
package com.sqisland.android.espresso.recycler_view_basic;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

public class TextViewHolder extends RecyclerView.ViewHolder {
  public TextView textView;
  public TextViewHolder(View itemView) {
    super(itemView);
    textView = (TextView) itemView;
  }
}

File: recycler-view-basic/app/src/main/java/com/sqisland/android/espresso/recycler_view_basic/TextViewHolder.kt
package com.sqisland.android.espresso.recycler_view_basic

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView

class TextViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
  val textView: TextView = itemView as TextView
}